ReactWindows/ReactNative.Shared/UIManager/UIViewOperationQueue.cs

```diff
@@ -27,6 +27,7 @@ public class UIViewOperationQueue
         private const long MaxNonBatchedTicksPerFrame = 83333;
 
         private static Stopwatch s_stopwatch = Stopwatch.StartNew();
+        private static readonly Action[] EMPTY_ACTIONS_ARRAY = new Action[0];
 
```

Comment:
Don't you need a new array, though, since it's going to start adding elements to it again?